app/renderer/js/console.js

```diff
@@ -0,0 +1,69 @@
+const NodeConsole = require('console').Console;
```

Comment:
Replace `NodeConsole` with `nodeConsole`.

Reply:
That would be a constructor?

Reply:
`NodeConsole` is a constructor that we later use by passing `WriteableStream` of file. Are you sure you want this capitalized because later we use it as `new NodeConsole()`.

Reply:
The linter also requires `NodeConsole` to be like as it is now.

Reply:
Hmm, make sense.